modules/tinymce/src/core/test/ts/browser/delete/TableDeleteTest.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable mocha/no-exclusive-tests */
```

Comment:
This shouldn't be here, I'm guessing it was left in by accident?

Reply:
Correct. I was debugging, and forgot to remove this one. Sorry about that.